apis/filters/parse.go

```diff
@@ -187,3 +187,32 @@ func (args Args) Validate(accepted map[string]bool) error {
 func FamiliarMatch(pattern string, ref string) (bool, error) {
 	return path.Match(pattern, ref)
 }
+
+// MatchKVList returns true if all the pairs in sources exist as key=value
+// pairs in the mapping at key, or if there are no values at key.
+func (args Args) MatchKVList(key string, sources map[string]string) bool {
```

Comment:
This part of code is similar with https://github.com/alibaba/pouch/blob/master/daemon/mgr/container_list.go#L74
Is there any plan to unify them?

Reply:
Yes, there's a plan, need to refactor to support a common filter package, as #2405.